tests/integration_tests/performance/test_network_tcp_throughput.py

```diff
@@ -0,0 +1,396 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the network throughput overhead added by Firecracker."""
+
+import itertools
+import json
+import logging
+import time
+import concurrent.futures
+import re
+import pytest
+from conftest import _test_images_s3_bucket
+from framework.artifacts import ArtifactCollection, ArtifactSet
+from framework.matrix import TestMatrix, TestContext
+from framework.builder import MicrovmBuilder
+from framework.statistics import core, consumer, producer, criteria, \
+    types, function
+from framework.utils import CpuMap, ProcessCpuAffinity, CmdBuilder, run_cmd, \
+    eager_map
+import host_tools.network as net_tools
+
+CONFIG = {
+    "time": 2,  # seconds
```

Comment:
time is too small, TCP takes some time to get to the best window size. I recommend at least 15s omitting first 4-5s.

Reply:
Raised the workload runtime to 20 seconds.